runtime/src/accounts_db.rs

```diff
@@ -17198,11 +17203,10 @@ pub mod tests {
                                     );
                                 });
 
-                                let storages = db.get_storages_for_slot(slot5).unwrap().to_vec();
+                                let storage = db.get_storage_for_slot(slot5).unwrap();
                                 let mut stored_accounts = Vec::default();
-                                assert_eq!(storages.len(), 1);
                                 let shrink_collect = db.shrink_collect(
```

Comment:
`shrink_collect` and other fns will be adapted soon to take a single append vec where appropriate.